google-apis-core/lib/google/apis/core/http_command.rb

```diff
@@ -168,7 +168,6 @@ def prepare!
             @form_encoded = true
             self.body = Addressable::URI.form_encode(url.query_values(Array))
             self.header['Content-Type'] = 'application/x-www-form-urlencoded; charset=UTF-8'
-            self.url.query_values = {}
```

Comment:
I'm not sure why we still have this. This was added [long ago](https://github.com/googleapis/google-api-ruby-client/commit/2c1fd125e9a166f6b47b90f4cd2c1a7412069953) and the code that actually used to update the URL with query values is [not there anymore](https://github.com/googleapis/google-api-ruby-client/commit/2c1fd125e9a166f6b47b90f4cd2c1a7412069953#diff-eb9a81517264e021c71f3df432a0558faea4eab4dfc11d98abffeb34b5017d47R275-R279)
This is causing issues while testing conformance tests. The query values are omitted in case of `put` or `post` methods.

Reply:
Agreed, I think this is a bug. Well found.